src/priorityQueue.js

```diff
@@ -0,0 +1,72 @@
+export class Element {
```

Comment:
I like the object oriented approach to the representation of data in the queue! ❤️